dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/util/json/DatasetVersionDTOTest.java

```diff
@@ -51,50 +50,40 @@ public void setUp() {
     public void tearDown() {
     }
 
-
     @Test
-    public void testReadDataSet() {
-        try {
+    public void testReadDataSet() throws IOException {
+        String filePath = "src/test/resources/json/JsonDatasetVersion.txt";
```

Comment:
This path should be relative, since if you package the application into distributed format (jar/war) the path will change. It should be done in similar manner to ExportServiceTest where the resources are also taken.

Reply:
Done.